### PR TITLE
Use gitignore.io for gitignore.

### PR DESCRIPTION
**Pull request by diminishedprime**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,90 @@
-node_modules
-/community-connectors/meetup/BUILD
```
Comment (Lucretiel) [inline .gitignore]: Probably you want to preserve this?

File: JSON-connect/.gitignore [hunk 2]
```diff
@@ -1,3 +0,0 @@
-node_modules
-package-lock.json
-.clasp.json
```
Comment (Lucretiel) [inline JSON-connect/.gitignore]: And these?

File: .gitignore (hunk 1 above)
Comment (diminishedprime) [reply .gitignore]: This actually shouldn't have been in there in the first place.

File: JSON-connect/.gitignore (hunk 2 above)
Comment (diminishedprime) [reply JSON-connect/.gitignore]: All of these are old and no-longer relevant.

**Pull request merged**